src/main/java/com/keniu/services/UserServiceImpl.java

```diff
@@ -13,9 +14,12 @@ public class UserServiceImpl implements UserService {
 
     private final UserRepository userRepository;
     private final UserMapper userMapper;
+    private final PasswordEncoder passwordEncoder;
 
     @Override
     public UserDto save(CreateUserRequestDto createUserRequestDto) {
+        createUserRequestDto.setPassword(
```

Comment:
Why do you set encode password to dto? 
It's better to map dto to User, set password and set ROLE_USER during registration

Reply:
At this stage, we do not receive roles from external sources. "During registration" -> in controller. I don't like the idea of encrypting password in controller

Reply:
I think we misunderstood each other. Don't need to encrypt password in controller of course. 
I mean when you create user it's better to add ROLE_USER.

Reply:
Added role to user